core/controllers/acl_decorators.py

```diff
@@ -3414,7 +3414,10 @@ def test_can_edit_entity(self, entity_type, entity_id, **kwargs):
             return can_edit_question(reduced_handler)(self, entity_id, **kwargs)
         elif entity_type == feconf.ENTITY_TYPE_TOPIC:
             return can_edit_topic(reduced_handler)(self, entity_id, **kwargs)
-        elif entity_type == feconf.ENTITY_TYPE_SKILL:
+        elif (
+            entity_type == feconf.ENTITY_TYPE_SKILL or
+            entity_type == feconf.IMAGE_CONTEXT_QUESTION_SUGGESTIONS
```

Comment:
This seems conceptually wrong to me. If I'm making suggestions to a question, shouldn't this relate to updating the question rather than updating the skill?

Reply:
I updated this one. Yes, it should be related to updating the question.